src/ImageNode.h

```diff
@@ -36,43 +31,35 @@ public slots:
     void warning(QString str);
     void fatal(QString str);
 protected:
-    bool loadImage(QString imagePath);
+    bool loadImage(QString file);
     ImageNode *m_p;
 protected slots:
     void onDestroyed();
 public:
     std::atomic<bool> m_ready{false};
```

Comment:
Remove `std::*`

Reply:
Be sure to keep this atomic though (until nodes are truly copied)